packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -7,7 +7,7 @@
 %global gem_name katello
 %global prerelease .pre.master
 %global mainver 3.13.0
-%global release 3
+%global release 5
```

Comment:
note comparing to the changelog, when release 4 was made this forgot to get bumped, so i'm skipping it here.  Wonder who did that! Such a rookie mistake! ;)